Parsers/ASimAuthentication/Parsers/ASimAuthenticationOktaOSS.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Authentication ASIM parser for Okta
-  Version: '0.2.3'
-  LastUpdated: Apr 04, 2024
+  Version: '0.3.0'
```

Comment:
did we test the new parsers with actual data?

Reply:
no data is there